js/util-galery.js

```diff
@@ -0,0 +1,68 @@
+
+const bigPhoto = document.querySelector('.big-picture');
+const body = document.querySelector('.body');
+const commentCounter = document.querySelector('.social__comment-count');
+const commentsLoader = document.querySelector('.comments-loader');
+const closeButton = document.querySelector('.big-picture__cancel  cancel');
+const socialComments = document.querySelector('.social__comments');
+
+const hideBigPhoto = () => {
+  bigPhoto.classList.add('hidden');
+  body.classList.remove('modal-open');
+  commentCounter.classList.remove('hidden');
+  commentsLoader.classList.remove('hidden');
+  //document.removeEventListener('keydown', onDocumentKeydown);
+  //closeButton.removeEventListener('click', onCloseButton);
+};
+
+const onDocumentKeydown = (evt) => {
+  if (evt.key === 'Escape') {
+    evt.preventDefault();
+    hideBigPhoto();
+  }
+};
+
+const onCloseButton = () => {
+  hideBigPhoto();
+};
+
+const renderPhotoDetails = ({url, description, likes, comments}) => {
+  bigPhoto.querySelector('.big-picture__img').src = url;
+  bigPhoto.querySelector('.social__caption').textContent = description;
+  bigPhoto.querySelector('.likes-count').textContent = likes;
+  document.querySelector('.comments-count').textContent = comments.length;
+};
+
+//сюда вставить рендер комментов
+const createComment = ({avatar, name, message}) => {
+  const comment = document.createElement('li');
```

Comment:
У комментария есть класс `social__comment`, нужно его добавить после создания li
```suggestion
  const comment = document.createElement('li');
  comment.classList.add('social__comment');
```